admin/load.php

```diff
@@ -141,7 +141,7 @@ function perflab_render_modules_page_field( $module_slug, $module_data, $module_
 			<?php echo esc_html( $module_data['name'] ); ?>
 		</legend>
 		<label for="<?php echo esc_attr( "{$base_id}_enabled" ); ?>">
-			<?php if ( $can_load_module && ! $is_standalone_plugin_loaded ) { ?>
+			<?php if ( ! is_wp_error( $can_load_module ) && ! $is_standalone_plugin_loaded ) { ?>
```

Comment:
This does not cater for false value. Would need to be:

```suggestion
			<?php if ( $can_load_module && ! is_wp_error( $can_load_module ) && ! $is_standalone_plugin_loaded ) { ?>
```